libraries/types/pbft_block/include/pbft/sync_block.hpp

```diff
@@ -28,6 +29,9 @@ class SyncBlock {
   std::vector<Transaction> transactions;
   bytes rlp() const;
   void clear();
+  bool hasEnoughValidCertVotes(size_t valid_sortition_players, size_t sortition_threshold, size_t pbft_2t_plus_1) const;
+
+  LOG_OBJECTS_DEFINE
```

Comment:
I do not like loggers inside small class that will be created N times, just necessary overhead. I think better solution is to throw exception and log it later